fs_brokers/apache_hdfs_broker/src/main/java/com/starrocks/broker/hdfs/FileSystemManager.java

```diff
@@ -361,6 +364,12 @@ public BrokerFileSystem getDistributedFileSystem(String scheme, String path, Map
                         Random random = new Random(currentTime);
                         int randNumber = random.nextInt(10000);
                         tmpFilePath = "/tmp/." + Long.toString(currentTime) + "_" + Integer.toString(randNumber);
+                        File tmpFile = new File(tmpFilePath);
```

Comment:
we shall respect the TMPDIR env var if set, otherwise use the default `/tmp` location.